dev/packages/alpha/system/0.0.1/kibana/visualization/4e4bb1e0-1b1b-11e7-b09e-037021c4f8df-ecs.json

```diff
@@ -0,0 +1,115 @@
+{
+    "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": {
+                "filter": [],
+                "query": {
+                    "language": "kuery",
+                    "query": ""
```

Comment:
Not for this PR, but I'm wondering if we could query here for the `stream.type` and `stream.dataset` and then this would query much less data.